### PR TITLE
Update virus-detection-in-spo.md

### PR DESCRIPTION
**Pull request by msbemba**
Adding Anti-malware and ransomware protection as a related artcile

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3832

File: microsoft-365/security/office-365-security/virus-detection-in-spo.md [hunk 1]
```diff
@@ -67,3 +67,8 @@ Microsoft 365 organizations that have [Microsoft Defender for Office 365](office
 ## More information
 
 For more information about anti-virus in SharePoint Online, OneDrive, and Microsoft Teams, see [Protect against threats](protect-against-threats.md) and [Turn on ATP for SharePoint, OneDrive, and Microsoft Teams](turn-on-atp-for-spo-odb-and-teams.md).
+
+
+## Related Artciles
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/virus-detection-in-spo.md]:
```suggestion
## Related Articles
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MSFTTracyP Please merge this PR. Thanks.

**Comment by yogkumgit** [comment]
Gentle reminder, @MSFTTracyP Please merge this PR. Thanks.

**Pull request merged**